### ISSUE TITLE
Is limited to total filesize <= 100MB

### ISSUE THREAD

**Issue by semuelle** [flex-dapps/enzypt#3]
Reason being the browser crypto library. Might be solved by splitting into multiple files or uploading as directory rather than ZIP.

**Comment by morelazers** [comment]
Worth noting this is an issue with the closed-source frontend codebase.

@aaronbarnardsound - should we open source the FE?

**Comment by aaronbarnardsound** [comment]
Yeah I think open sourcing the front end is a good option. Would like to get it in better shape beforehand though.
